plugins/editor/palettes/NodePalette.qml

```diff
@@ -11,7 +11,6 @@ CodePalette{
     id: palette
     type: "qml/Object"
 
-    property var editingFragment: null
     property var documentHandler: null

```

Comment:
`documentHandler` is easy to capture from `editor`

At the start of each function where it's used, you can:

```js
var documentHandler = editor.documentHandler
```